src/targets/intents/index.ejs

```diff
@@ -13,7 +13,8 @@
 {{.ThemeCSS}}
 <% if (__STACK_ASSETS__) { %>
 {{.CozyClientJS}}
-<% } else { %>
+<% } %>
+<% if (__TARGET__ !== 'mobile') { %>
```

Comment:
Well, I don't think we already have a mobile target, but let's be future-proof!

Reply:
Ha ha you're right, I just reverted without diving any further into those kind of considerations 🤓